tests/components/alexa/test_smart_home.py

```diff
@@ -1118,7 +1135,7 @@ def assert_equal(self, namespace, name, value):
         for prop in self.properties:
             if prop['namespace'] == namespace and prop['name'] == name:
                 assert prop['value'] == value
-            return prop
+                return prop
```

Comment:
I think this was a mistake, the assert would always exit with a success if the first property was not the one being asserted. This fixes that so all properties are checked and at least one must match the assertion.